hack/build_lua_deps.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DESTDIR=./rootfs/etc/nginx/lua/vendor
+
+get_src()
+{
+  hash="$1"
+  url="$2"
+  f=$(basename "$url")
+
+  curl -sSL "$url" -o "$f"
+  tar xzf "$f"
+  rm -rf "$f"
+}
+
+cd "$DESTDIR"
```

Comment:
By doing this you're leaving a lot of unnecessary files in the destdir. Instead you can create a temporary folder(`BUILD_PATH`) and work inside that folder. `make install` will take care of installing the files into the correct target folder if you export `DESTDIR`